x-pack/plugins/security_solution/public/timelines/components/row_renderers_browser/catalog/index.tsx

```diff
@@ -40,6 +40,26 @@ const Link = ({ children, url }: { children: React.ReactNode; url: string }) =>
   </EuiLink>
 );
 
+export const eventRendererNames: { [key in RowRendererId]: string } = {
+  [RowRendererId.alerts]: i18n.ALERTS_NAME,
+  [RowRendererId.auditd]: i18n.AUDITD_NAME,
+  [RowRendererId.auditd_file]: i18n.AUDITD_FILE_NAME,
+  [RowRendererId.library]: i18n.LIBRARY_NAME,
+  [RowRendererId.system_security_event]: i18n.AUTHENTICATION_NAME,
+  [RowRendererId.system_dns]: i18n.DNS_NAME,
+  [RowRendererId.netflow]: i18n.FLOW_NAME,
+  [RowRendererId.system]: i18n.SYSTEM_NAME,
+  [RowRendererId.system_endgame_process]: i18n.PROCESS,
+  [RowRendererId.registry]: i18n.REGISTRY_NAME,
+  [RowRendererId.system_fim]: i18n.FIM_NAME,
+  [RowRendererId.system_file]: i18n.FILE_NAME,
+  [RowRendererId.system_socket]: i18n.SOCKET_NAME,
+  [RowRendererId.suricata]: 'Suricata',
+  [RowRendererId.threat_match]: i18n.THREAT_MATCH_NAME,
+  [RowRendererId.zeek]: i18n.ZEEK_NAME,
+  [RowRendererId.plain]: '',
```

Comment:
I am struggling to find a scenario where we would display the plain RowRenderer. I think that it won't be displayed on the alert page. 

When I call `getRowRenderer` and no `ronRenderer` is found it returns null Instead of returning `plainRowRenderer`.

I couldn't find any reference to  [plain_row_renderer.tsx](https://github.com/elastic/kibana/blob/3612a7a300e0a08d2537104b986508cdd34fe102/x-pack/plugins/timelines/public/components/t_grid/body/renderers/plain_row_renderer.tsx) on the code.

[Here](https://github.com/elastic/kibana/blob/4fa3dc46cb14f041d21b1b6a06961eb490602701/x-pack/plugins/security_solution/public/timelines/components/timeline/body/renderers/index.ts#L25
) there is a comment mentioning it but the comment is outdated because `plainRowRenderer` is not included on `defaultRowRenderers`
<img width="567" alt="Screenshot 2021-08-12 at 14 13 54" src="https://user-images.githubusercontent.com/1490444/129194897-ab345724-91f3-484c-9a2b-735cb58b1712.png">

If I am right, we could delete `plain_row_renderer.tsx`